CONTRIBUTING.md

```diff
@@ -25,15 +25,15 @@ git clone git@github.com:mapbox/mapbox-gl-js.git
 
 Install node module dependencies
 ```bash
-cd mapbox-gl-js &&
 yarn install
 ```
 
 ### Linux
 
-Install [git](https://git-scm.com/), [node.js](https://nodejs.org/) (version ^10.15), [GNU Make](http://www.gnu.org/software/make/), and libglew-dev
+Install [git](https://git-scm.com/), [node.js](https://nodejs.org/) version 14, [GNU Make](http://www.gnu.org/software/make/), and libglew-dev
 ```bash
-sudo apt-get update &&
+sudo apt-get update
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
```

Comment:
Works for me, but is nodesource the place to acquire node@14? I thought they were a very specific enterprise flavor of node.

Reply:
I'm not sure, I've setup a ubuntu machine and this is what worked for me. I can look at alternatives

Reply:
100% fine with me not to bother. It's linux-specific, so if this is the way you do it there and it works, it's 💯 okay with me. (nodesource is great, I'm only just familiar with them as an enterprise node company)